### ISSUE TITLE
Registration - Accessibility: CGU checkbox is not accessible by Voiceover

### ISSUE THREAD

**Issue by giomfo** [dinsic-pim/tchap-ios#194]
Problème avec Voiceover, IOS: Impossible d'accepter les conditions d'utilisation lors d'une création de compte
